### DISCUSSION TITLE
Unable to lock file, Tiling images failed! Error in geometric module

### DISCUSSION THREAD

**Discussion by dkczk** [davidfrantz/force#213] [Q&A] [answered]
Hey guys,
I processed Landsat 8/9 Images and went with a different projection in the parameter file (EPSG: 4326). I have adjusted the affected parameters as follows:
```
TILE_SIZE = 0.9
BLOCK_SIZE = 0.3
RESOLUTION_LANDSAT = 0.0003
PROJECTION = GEOGCS["GCS_WGS_1984",DATUM["D_WGS_1984",SPHEROID["WGS_1984",6378137,298.257223563]],PRIMEM["Greenwich",0],UNIT["Degree",0.017453292519943295]]
```
The processing worked as far as I can tell. The only thing missing are the quicklook JPGs. Those seem to be incomplete and are named e.g. "20220720_LEVEL2_LND08_OVV.jpg.lock". So I looked up the log-files which basically all say:
> PATH/TO/FILE/LC08_L1TP_191024_20220805_20220818_02_T1: dc:  64.90%. wc:   0.30%. sc:  16.17%. cc:  54.52%. AOD: 0.1679. > # of targets: 276/3558. number of cols must be > 0.
> number of rows must be > 0.
> number of cols must be > 0.
> number of rows must be > 0.
> number of cols must be > 0.
> number of rows must be > 0.
> Unable to lock file PATH/TO/LEVEL2/X0006_Y0002/20220805_LEVEL2_LND08_OVV.jpg (timeout: 63.614139s, nx/ny: 0/0). Unable to > lock file PATH/TO/LEVEL2/X0007_Y0002/20220805_LEVEL2_LND08_OVV.jpg (timeout: 63.614139s, nx/ny: 0/0). error in cubing Level 2 products
> Tiling images failed! Error in geometric module.

What is the cause of the error? Did I anything wrong when adjusting the projection parameters in the parameter file?
Thanks in advance and best wishes.

**Comment by davidfrantz** [comment]
Hi @dkczk,

it might be that this is related to the geographic projection where the resolution is specified with fractional values. There is probably some rogue integer/float casting that causes this issue. I will investigate as soon as possible.

Best,
David

**Comment by davidfrantz** [comment, marked as answer]
I move this discussion to issues and close this one. 

Please follow here:
https://github.com/davidfrantz/force/issues/214
